pomegranate/FactorGraph.pyx

```diff
@@ -395,7 +395,7 @@ cdef class FactorGraph( GraphModel ):
 			"states": [json.loads(state.to_json()) for state in self.states],
 			"edges": self.edges
 		}
-		return json.dumps(model, separators=separators, indent=indent)
+		return json.dumps(model, separators=separators, indent=indent, sort_keys=True)
```

Comment:
I am not sure if you would like me to send another patch specifically for this -- I am happy to do that, just let me know.